ballerina-command/src/main/java/org/ballerinalang/command/cmd/ListCommand.java

```diff
@@ -101,6 +102,7 @@ private static void listDistributions(PrintStream outStream) {
             listOfFiles = folder.listFiles();
             outStream.println("Distributions available locally: \n");
             List<String> installedVersions = new ArrayList<>();
+            Arrays.sort(listOfFiles);
```

Comment:
Shouldn't we show the latest at the top? 

And are remote version always sorted?

Reply:
@MaryamZi  Yes sorting descending is better. Currently remote versions are also sorted ascending order, lets handle that logic in the server side